package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A basic webpack project setup, packed with all kinds of useful goodies.",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --watch --config config/webpack/dev.config.js",
+    "start": "webpack-dev-server --watch --hot --inline --config config/webpack/dev.config.js",
```

Comment:
The `--hot` and `--inline` arguments should be redundant:
- Hot Module Replacement is already activated via the webpack `dev.config.js`, see: https://github.com/voodoods/webpack-boilerplate/blob/master/config/webpack/dev.config.js#L11
- Since we use Webpack 4, _inline mode_ is enabled per default, see also: https://webpack.js.org/configuration/dev-server/#devserver-inline

Reply:
Originally I got `[HMR] Hot Module Replacement is disabled.` logged to the console. In https://github.com/webpack/webpack/issues/1151 it was mentioned that you still have to pass `--hot` as a command line argument for it to work which for me is also the case on two different machines.

`--inline` does seem to be unnecessary though.